pages/dkp/kommander/2.2/dkp-upgrade/index.md

```diff
@@ -1,10 +1,38 @@
 ---
 layout: layout.pug
-navigationTitle: DKP upgrade
-title: How to upgrade DKP
+navigationTitle: DKP Upgrade
+title: How to upgrade DKP via CLI
 menuWeight: 8
 excerpt: Upgrade DKP to the latest version and benefit from the full DKP experience.
 beta: false
 ---
 
 This section describes how to upgrade your environment to the latest version of DKP.
+
+The upgrade process represents an important step of your environment's lifecycle, as it ensures that you benefit from the latest improvements, enhanced management of clusters and better performance.
+
+When upgrading DKP, you must follow several steps that differ depending on your setup. The following indications specify which steps to follow if you are running a stand-alone cluster (Management cluster) or a multi-cluster environment (Management Cluster with Attached or Managed Clusters).
+
+<p class="message--warning"><strong>WARNING: </strong>Upgrade your Platform and Catalog Applications before upgrading Konvoy and Kubernetes. Some applications of the previous release do not support Kubernetes 1.22, and upgrading Kubernetes to 1.22 is be part of the Konvoy upgrade.
+</p>
+
+## Understand the upgrade process
+
+The overall process for upgrading to the latest version of DKP has the following high-level steps:
+
+1.  [Upgrade Kommander][upgrade-kommander]. This takes place in your Management Cluster and includes the upgrade of your Platform Applications in that cluster.
+
+1.  [Upgrade your Workspaces][upgrade-workspaces] on a per-Workspace basis, if your environment has additional Workspaces (with Managed or Attached clusters).
```

Comment:
This and the next step technically are valid only for Enterprise users (Essential does not include multicluster). Does this mean we would want separate upgrade pages for Essential and Enterprise? @cmastr

Reply:
Hi! I see two options here: 
A. I add text at the beginning of that apply steps for multi-cluster environments specifying this only for Enterprise only, like this:
```suggestion
1.  For Enterprise customers only: [Upgrade your Workspaces][upgrade_workspaces] on a per-Workspace basis.
```
B. I leave this page as it is including all steps and add the Enterprise tag on top. Then, I create a copy of this document for Essential and delete these steps. 
I like B better, but I am not sure if we want to do that yet, as we were planning the major restructuring of our site for 2.3?
@cmastr I leave the decision up to you, since you have the picture more in mind than I do :)